pkgs/by-name/ce/celeste-classic/package.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenvNoCC, fetchzip, autoPatchelfHook, pkgs, lib, practiceMod ? false }:
+
+let
+  folder = if practiceMod then "CELESTE*Practice*" else "CELESTE";
```

Comment:
small nitpick: Linux people usually prefer "directory" to "folder" (you can rename the var to `dir`)